js/codemeta_generation.js

```diff
@@ -312,21 +312,72 @@ function importShortOrg(fieldName, doc) {
     }
 }
 
+function importReview(doc) {
+    if (doc !== undefined) {
+        directReviewCodemetaFields.forEach(item => {
+            setIfDefined('#' + item, doc[item]);
+        });
+    }
+}
+
+function authorsEqual(author1, author2) {
+    // TODO should test more properties for equality?
+    return author1.givenName === author2.givenName
+        && author1.familyName === author2.familyName
+        && author1.email === author2.email;
```

Comment:
The `id` is what matters in JSON-LD. Without it, JSON-LD says we should consider them different nodes. (see also #42, which doesn't need to be fixed before merging this PR)

However, given that these are the only ways we display an author, I think it's fine to have these three equalities as a fallback when both nodes are missing an id.

Reply:
Ok, I'm looking into #42 !